book/src/drivers.md

```diff
@@ -24,11 +24,12 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Datera](https://github.com/Datera/datera-csi) | `dsp.csi.daterainc.io` | v1.0 | A Container Storage Interface (CSI) Driver for Datera Data Services Platform (DSP) | Persistent | Read/Write Single Pod | Yes |Snapshot
-[Dell EMC Isilon](https://github.com/dell/csi-isilon) | `csi-isilon.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC Isilon](https://www.dellemc.com/en-us/storage/isilon/index.htm) | Persistent | Read/Write Multiple Pods | Yes | Snapshot
-[Dell EMC PowerMax](https://github.com/dell/csi-powermax) | `csi-powermax.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC PowerMax](https://www.dellemc.com/en-us/storage/powermax.htm) | Persistent | Read/Write Single Pod | Yes |
-[Dell EMC Unity](https://github.com/dell/csi-unity) | `csi-unity.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC Unity](https://www.dellemc.com/en-us/storage/unity.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
-[Dell EMC VxFlexOS](https://github.com/dell/csi-vxflexos) | `csi-vxflexos.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for Dell EMC VxFlexOS | Persistent | Read/Write Single Pod | Yes | Snapshot
-[Dell EMC XtremIO](https://github.com/dell/csi-xtremio-deploy) | `csi-xtremio.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC XtremIO](https://www.dellemc.com/en-us/storage/xtremio-all-flash.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Dell EMC Isilon](https://github.com/dell/csi-isilon) | `csi-isilon.dellemc.com` | v1.1 | A Container Storage Interface (CSI) Driver for [Dell EMC Isilon](https://www.delltechnologies.com/en-us/storage/isilon/index.htm) | Persistent | Read/Write Multiple Pods | Yes | Snapshot
+[Dell EMC PowerMax](https://github.com/dell/csi-powermax) | `csi-powermax.dellemc.com` | v1.1 | A Container Storage Interface (CSI) Driver for [Dell EMC PowerMax](https://www.delltechnologies.com/en-us/storage/powermax.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Dell EMC PowerStore](https://github.com/dell/csi-powerstore) | `csi-powerstore.dellemc.com` | v1.1 | A Container Storage Interface (CSI) Driver for [Dell EMC PowerStore](https://www.delltechnologies.com/en-us/storage/powerstore-storage-appliance.htm) | Persistent | Read/Write Single Pod | Yes | 
+[Dell EMC Unity](https://github.com/dell/csi-unity) | `csi-unity.dellemc.com` | v1.1 | A Container Storage Interface (CSI) Driver for [Dell EMC Unity](https://www.delltechnologies.com/en-us/storage/unity.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Dell EMC VxFlexOS](https://github.com/dell/csi-vxflexos) | `csi-vxflexos.dellemc.com` | v1.1 | A Container Storage Interface (CSI) Driver for Dell EMC VxFlexOS | Persistent | Read/Write Single Pod | Yes | Snapshot
```

Comment:
For all other Dell EMC drivers, you have provided a corresponding product page.  You don't have a similar one for VxFlexOS?

Reply:
Good catch ; it is fixed :-)